src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts

```diff
@@ -50,19 +50,21 @@ export namespace DirectionsQuestionnaire {
         selfWitness: directionsQuestionnaire.selfWitness.option.option as YesNoOption,
         noOfOtherWitness: directionsQuestionnaire.otherWitnesses ? directionsQuestionnaire.otherWitnesses.howMany : undefined
       },
-      expertReports: directionsQuestionnaire.expertReports && directionsQuestionnaire.expertReports.rows
-        && directionsQuestionnaire.expertReports.rows.map(row => ({
+      expertReports: (!(directionsQuestionnaire.expertReports.rows.length === 1
+        && directionsQuestionnaire.expertReports.rows[0].isEmpty())) ?
```

Comment:
This smells like a hack. If possible, it would be better to ensure the `expertReports` is populated only with valid data when it's defined by the user.

Reply:
i completely understand what you are saying... I thought the same. ExpertReports uses common code to deserializeRows which is used in other places in the app where ever we have multiple rows. Not sure why its originally developed in a way to generate empty but it does generate empty object when there are no rows. I decided not to change that thinking its probably like that for some valid reason. So, my other option is change how deserialise is done in expertRows, I could do that but thought this is okay!!

Reply:
actually i am not sure why backend is like this 
` @Valid
    @Size(min = 1, max = 20)
    private final List<ExpertReport> expertReports;`